app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post, counter_cache: :likes_counter
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
+end
```

Comment:
- [ **OPTIONAL** ] As I mentioned in the previous comment and for best practices, I think it would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method `private`. **You can use the same example like the one from the comment above**